py3status/modules/rss_aggregator.py

```diff
@@ -58,15 +74,18 @@ class Py3status:
 
     def post_config_hook(self):
         self._cached = "?"
-        if self.aggregator not in ['owncloud']:  # more options coming
+        if self.aggregator not in ['owncloud', 'ttrss']:  # more options coming
             raise ValueError('%s is not a supported feed aggregator' % self.aggregator)
         if self.user is None or self.password is None:
             raise ValueError("user and password must be provided")
 
     def check_news(self):
         if self.aggregator == "owncloud":
             rss_count = self._get_count_owncloud()
-            self._cached = rss_count or self._cached
+        elif self.aggregator == "ttrss":
+            rss_count = self._get_count_ttrss()
+
+        self._cached = self._cached if rss_count is None else rss_count
```

Comment:
you can write this as 
`self._cached =  rss_count or self._cached`
which I find more readable